docs/sources/reference/api.md

```diff
@@ -1,69 +1,81 @@
 ---
-title: Grafana Loki HTTP API
-menuTitle: "HTTP API"
-description: Loki exposes REST endpoints for operating on a Loki cluster. This section details the REST endpoints.
+title: Loki HTTP API
+menuTitle: HTTP API
+description: Loki exposes HTTP endpoints for data ingestion, data retrieval, as well for cluster management.
 aliases:
 - ../api/
 weight: 100
 ---
 
-# Grafana Loki HTTP API
+# Loki HTTP API
 
-Grafana Loki exposes an HTTP API for pushing, querying, and tailing log data.
-Note that authenticating against the API is
-out of scope for Loki.
+Loki exposes an HTTP API for pushing, querying, and tailing log data, as well
+as for viewing and managing cluster information.
 
-## Microservices mode
+**Note that authorization is not part of the Loki API.**
+Authorization needs to be done separately, e.g. using an open-source load-balancer such as NGINX.
 
-When deploying Loki in microservices mode, the set of endpoints exposed by each
-component is different.
+## Endpoints
 
-These endpoints are exposed by all components:
+### Ingest endpoints
 
-- [`GET /ready`](#identify-ready-loki-instance)
-- [`GET /log_level`](#change-log-level-at-runtime)
-- [`GET /metrics`](#return-exposed-prometheus-metrics)
-- [`GET /config`](#list-current-configuration)
+These endpoints are exposed by the `distributor`, `write`, and `all` components:
+
+- [`POST /loki/api/v1/push`](#ingest-logs)
+
+A [list of clients]({{< relref "../send-data" >}}) can be found in the clients documentation.
+
+### Query endpoints
+
+These HTTP endpoints are exposed by the `querier`, `query-frontend`, `read`, and `all` components:
+
+- [`GET /loki/api/v1/query`](#query-logs-at-a-single-point-in-time)
+- [`GET /loki/api/v1/query_range`](#query-logs-within-a-range-of-time)
+- [`GET /loki/api/v1/labels`](#query-labels)
+- [`GET /loki/api/v1/label/<name>/values`](#query-label-values)
+- [`GET /loki/api/v1/series`](#query-streams)
+- [`GET /loki/api/v1/index/stats`](#query-log-statistics)
+- [`GET /loki/api/v1/index/volume`](#query-log-volume)
+- [`GET /loki/api/v1/index/volume_range`](#query-log-volume)
+- [`GET /loki/api/v1/tail`](#stream-logs)
+
+API endpoints starting with `/api/prom` are [Prometheus API-compatible](https://prometheus.io/docs/prometheus/latest/querying/api/) and the result formats can be used interchangeably.
```

Comment:
I actually removed the `/api/prom/*` endpoints from the list of query endpoints because they are deprecated (we don't remove them from code, only from docs). I didn't see that we still also use `/api/prom/*` endpoints in the ruler API.
@dannykopping Do you think we can also remove them from docs?

Reply:
Or should we remove the note about Prometheus compatibility completely? I don't see a point in that anyway.

Reply:
I think these endpoints are still used extensively by the Grafana Alerting UI and [`cortex-tool`](https://github.com/grafana/cortex-tools/blob/1499f5fb286e672cd238a525169d4cb05f964eab/pkg/commands/rules.go#L245).